src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -161,12 +161,14 @@ class WorkflowManagerActor(backend: Backend) extends Actor with CromwellActor {
   }
 
   private def callStdoutStderr(workflowId: WorkflowId, callFqn: String): Future[Any] = {
+    def callKey(descriptor: WorkflowDescriptor, callName: String, key: ExecutionDatabaseKey) =
+      CallKey(descriptor.namespace.workflow.calls.find(_.unqualifiedName == callName).get, key.index)
```

Comment:
Sounds good.  Do you think it's sufficiently clear the parameter would represent an unqualified name?